src/libraries/System.Runtime/tests/System/Reflection/ReflectionCacheTests.cs

```diff
@@ -33,24 +76,125 @@ public void InvokeClearCache_NoExceptions()
         }
 
         [ActiveIssue("https://github.com/dotnet/runtime/issues/50978", TestRuntimes.Mono)]
-        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsMetadataUpdateSupported))]
```

Comment:
Changed the `Theory` to `Fact` because of getting exception with `RemoteExecutor` and could not find how to fix
```log
Assert.All() Failure: 1 out of 1 items in the collection did not pass.
[0]: Item: Boolean justSpecificType
     Xunit.Sdk.TrueException: Field marshaling is not supported by Invoke: justSpecificType
     Expected: True
     Actual:   False
        at Xunit.Assert.True(Nullable`1 condition, String userMessage) in /_/src/xunit.assert/Asserts/BooleanAsserts.cs:line 132
        at Xunit.Assert.True(Boolean condition, String userMessage) in /_/src/xunit.assert/Asserts/BooleanAsserts.cs:line 116
        at Microsoft.DotNet.RemoteExecutor.RemoteExecutor.<>c.<GetMethodInfo>b__37_0(FieldInfo fi) in /_/src/Microsoft.DotNet.RemoteExecutor/src/RemoteExecutor.cs:line 504
        at Xunit.Assert.<>c__DisplayClass11_0`1.<All>b__0(T item, Int32 index) in /_/src/xunit.assert/Asserts/CollectionAsserts.cs:line 34
        at Xunit.Assert.All[T](IEnumerable`1 collection, Action`2 action) in /_/src/xunit.assert/Asserts/CollectionAsserts.cs:line 61
```